### PR TITLE
provide 'fetch()' response clone in metrics, and introduce 'WorkerMetric'

### PR DESCRIPTION
**Pull request by jwerle**

### CODE REVIEW

File: api/diagnostics/window.js [hunk 1]
```diff
@@ -175,10 +176,37 @@ export class XMLHttpRequestMetric extends Metric {
...
 }
 
+export class WorkerMetric extends Metric {
+  static GlobalWorker = globalThis.Worker
+
+  constructor (options) {
+    super()
+    const self = this
+    this.channel = dc.channel('Worker')
+    this.Worker = class Worker extends globalThis.Worker {
```
Comment (heapwolf) [inline api/diagnostics/window.js:186]: 😎

**Review by heapwolf — APPROVED** [review]

**Review by mribbons — CHANGES_REQUESTED** [review]
I know it's not related to this branch specifically, but:
cli.cc:768:10: error: use of undeclared identifier 'SIGUSR1'

Also, fix linting :)

**Review by chicoxyzzy — APPROVED** [review]
👍

File: api/diagnostics/window.js [hunk 2]
```diff
@@ -175,10 +176,37 @@ export class XMLHttpRequestMetric extends Metric {
   }
 }
 
+export class WorkerMetric extends Metric {
+  static GlobalWorker = globalThis.Worker
+
+  constructor (options) {
+    super()
+    const self = this
```
Comment (chicoxyzzy) [inline api/diagnostics/window.js:184]: linter is not happy that `self` is not used after the variable initialization

**Pull request merged**